src/object.rs

```diff
@@ -2570,3 +2429,116 @@ impl<'a> BindingBuilder<'a> {
         }
     }
 }
+
+#[repr(C)]
```

Comment:
```suggestion
#[repr(transparent)]
```

Because that's how you use it :)